### PR TITLE
Feature/us 029 add validations to put operation

### PR DESCRIPTION
**Pull request by Gra2411**
Here are the validations to the PUT operation to update a user.

### CODE REVIEW

File: src/main/java/com/agilethought/atsceapi/service/UserServiceImpl.java [hunk 1]
```diff
@@ -68,39 +74,30 @@ public void deleteUserById(String id) {
...
-		}else {
-			throw new NotFoundException("User Not Found with id: " + Id);
-		}
+		setUpperCase(user);
+		User updatedUser = userRepository.save(user);
+		return orikaMapperFacade.map(updatedUser, UpdateUserResponse.class);
+	}
 
+	private static void setUpperCase(User request){
+		request.setFirstName(request.getFirstName().toUpperCase());
```
Comment (ivanflores15) [inline src/main/java/com/agilethought/atsceapi/service/UserServiceImpl.java:100]: I liked the way to set attributes by reference. Good job

File: src/main/java/com/agilethought/atsceapi/validator/UserDataValidationUtils.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+import com.agilethought.atsceapi.exception.BadRequestException;
+import org.springframework.stereotype.Service;
+
+import static com.agilethought.atsceapi.exception.ErrorMessage.*;
+import static com.agilethought.atsceapi.exception.ErrorMessage.BAD_REQUEST_MESSAGE_TYPE;
+import static com.agilethought.atsceapi.validator.ValidationUtils.*;
+
+@Service
+public class UserDataValidationUtils {
```
Comment (ivanflores15) [inline src/main/java/com/agilethought/atsceapi/validator/UserDataValidationUtils.java:11]: Good approach to valide commons field and avoid duplication of code!

**Review by ivanflores15 — APPROVED** [review]

**Review by luisgalindoluna — APPROVED** [review]
Looks good, approved

**Review by EmiliaGuillen-dev — APPROVED** [review]
looks good

**Pull request merged**